packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/interfaces/firestore_platform.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:async';
+
+import 'package:meta/meta.dart' show required;
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+import './document_reference_platform.dart';
+import './query_platform.dart';
+import './transaction_platform.dart';
+import './write_batch_platform.dart';
+
+import '../implementations/method_channel_firestore.dart';
+
+/// The Firestore platform interface.
+abstract class FirestorePlatform extends PlatformInterface {
+  /// Constructor
+  FirestorePlatform() : super(token: _token);
+
+  static const Object _token = Object();
```

Comment:
Make these final, this was an overlooked bug, see: https://github.com/flutter/plugins/pull/2417

Reply:
Ouch! Thanks for the heads up, I'll make all my `_token`s `final` instead of `const`